### PR TITLE
https://wso2.org/jira/browse/PC-28

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: modules/extensions/process/publisher/process/asset.js [hunk 1]
```diff
@@ -80,7 +80,9 @@ asset.renderer = function(ctx) {
             var log = new Log();
 
             var resourcePath = page.assets.path;
-            log.info(resourcePath);
+            if(log.isDebugEnabled()){
```
Comment (vinok88) [inline modules/extensions/process/publisher/process/asset.js]: format code.

File: modules/extensions/process/publisher/process/apis/delete_Predecessor.jag [hunk 2]
```diff
@@ -2,11 +2,15 @@
 importPackage(org.wso2.carbon.pc.core);
 
 var log = new Log();
-log.info("Predecessor delete logic");
+if(log.isDebugEnabled()){
+    log.debug("Predecessor delete logic");
+}
```
Comment (chathurace) [inline modules/extensions/process/publisher/process/apis/delete_Predecessor.jag]: Make log messages more descriptive. Also write them in the form "Creating predecessors of process X".

Comment (isurusuranga) [inline modules/extensions/process/publisher/process/apis/delete_Predecessor.jag]: Fixed in https://github.com/wso2/product-pc/pull/34

File: modules/extensions/process/publisher/process/asset.js (hunk 1 above)
Comment (isurusuranga) [inline modules/extensions/process/publisher/process/asset.js]: Already fixed

**Pull request merged**